RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -407,10 +409,64 @@ int remainingEmbers()
 
 void auto_buyFromSeptEmberStore()
 {
+	if(!auto_haveSeptEmberCenser())
+	{
+		return;
+	}
 	if(remainingEmbers() == 0)
 	{
 		return;
 	}
+
+	// mouthwash for leveling
+	item mouthwash = $item[Mmm-brr! brand mouthwash];
+	boolean disregard_karma = get_property("auto_disregardInstantKarma").to_boolean();
+	auto_openMcLargeHugeSkis(); // make sure our skis are open for cold res
+	for (int imw = 0 ; imw < 3 ; imw++) // We can use up to 3 mouthwash
+	{
+		// If we have at least 4 embers remaining, don't overlevel, they can be used for something else
+		boolean happy_to_overlevel = disregard_karma && remainingEmbers() < 4;
+		boolean want_to_mouthwash_level = (my_level() < 13 || happy_to_overlevel) && my_level()<15;
+		// Even disregarding karma, never level above 15 using mouthwash as a sanity limit
+		want_to_mouthwash_level = want_to_mouthwash_level && my_level()<15;
```

Comment:
no need because you already check level < 15 above

Reply:
Done.